policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml

```diff
@@ -3,59 +3,88 @@ SINGLE:
   0:
     2021-01-01: 2_100
     2022-01-01: 2_200
+    2023-01-01: 2_300
+    
```

Comment:
I tried to add uprating to each of these, but it causes error saying "duplicate metadata"

Reply:
Does a structure like above work?

Reply:
Yes, it's working.